coresimd/src/x86/i586/sse.rs

```diff
@@ -1688,18 +1688,15 @@ pub unsafe fn _mm_stream_ps(mem_addr: *mut f32, a: f32x4) {
 /// memory hint.
 #[inline(always)]
 #[target_feature = "+sse"]
-// generates movnti on i686 and x86_64 but just a mov on i586
-#[cfg_attr(all(test,
-               any(target_arch = "x86_64",
-                   all(target_arch = "x86", target_feature = "sse2"))),
-           assert_instr(movnti))]
-pub unsafe fn _mm_stream_pi(mem_addr: *mut i64, a: i64) {
+// #[cfg_attr(test, assert_instr(movntq))] // FIXME: llvm codegens wrong instr
+pub unsafe fn _mm_stream_pi(mem_addr: *mut i8x8, a: i8x8) {
```

Comment:
We should be using `_m64` here (which might probably fix the FIXME since that is the magic `x86_mmx` type).